plugins/org.jkiss.dbeaver.ext.altibase.ui/src/org/jkiss/dbeaver/ext/altibase/ui/views/CreateTypesetPage.java

```diff
@@ -87,4 +88,9 @@ public String getProcedureName() {
     public boolean isPageComplete() {
         return !CommonUtils.isEmpty(name);
     }
+
+	@Override
+	public DBSObject getObject() {
+		return null;

```

Comment:
I think it is better to return the `procedure` here.

Reply:
@LonwoLonwo, 

Thanks for your tip :)

I updated it according to your recommendation.